lte/gateway/c/core/oai/test/mme_app_task/BUILD.bazel

```diff
@@ -105,6 +105,16 @@ cc_test(
     ],
 )
 
+cc_library(
+    name = "mme_procedure_test_fixture",
+    srcs = ["mme_procedure_test_fixture.h"],
+    deps = [
+        ":mme_app_test_core",
+        "//lte/gateway/c/core/oai/test/mock_tasks",
+        "@com_google_googletest//:gtest_main",
```

Comment:
Is gtest actually used in this header? I don't see it imported.

If gtest is needed - we do not need the main-auto-wrapped import I think. So we could change this line to `"@com_google_googletest//:gtest",`.

Reply:
I think it is needed to pull in `::testing::Test`, but will double check. But I did not realize there was a distinction between gtest and gtest_main :D Will double check to see what is needed. Thanks!

Reply:
#include <gtest/gtest.h> is already included. Testing with "@com_google_googletest//:gtest" seems to work.

Reply:
> #include <gtest/gtest.h> is already included

Aah ok, I should have ctr-f'd.

Reply:
Yeah I think gtest is needed, testing this is a little bit of an effort because of how C++ gobbles includes :P but I *think* it is needed